web/cobrands/northamptonshire/assets.js

```diff
@@ -376,6 +376,10 @@ var northants_defaults = $.extend(true, {}, fixmystreet.assets.alloy_defaults, {
   select_action: true,
   actions: {
     asset_found: function(asset) {
+      var emergency_state = ncc_is_emergency_category();
+      if (emergency_state.relevant && !emergency_state.body) {
```

Comment:
The body checks are required because if you are a staff user then none of the emergency hide/show code fires so if you click off a road and then select a Non emergency category, then an emergency category then move the pin to a road the form is not re-enabled.

Reply:
Hmm, I see, but I do find it confusing. Would it be better if `ncc_is_emergency_category` was renamed `ncc_ignore_emergency_message` and returned only `relevant === currently_shown || body` - then all this code would only have to check that boolean?

Reply:
Yeah, it's a bit sub-optimal but as you say the original code needs to make distinctions.